mulighetsrommet-arena-ords-proxy/src/main/kotlin/no/nav/mulighetsrommet/arena_ords_proxy/ArenaOrdsClient.kt

```diff
@@ -36,12 +33,11 @@ class ArenaOrdsClient(val config: ArenaOrdsConfig) {
         }
     }
 
-    @OptIn(InternalAPI::class)
     suspend fun getFnrByArenaPersonId(arenaPersonIdList: ArenaPersonIdList): ArenaPersonIdList {
         val response = client.request("/arena/api/v1/person/identListe") {
             contentType(ContentType.Application.Json)
             method = HttpMethod.Post
-            body = Json.encodeToString(ArenaPersonIdList)
+            setBody(Json.encodeToString(arenaPersonIdList))
```

Comment:
Shit, denne var ny for meg. Må ha kommet med i Ktor 2 ser jeg ifølge dokumentasjonen.